src/axom/core/tests/core_array.hpp

```diff
@@ -1931,4 +1931,18 @@ TEST(core_array, checkVariadicCtors)
   Array<int, 3> arr12(s, s, s);
 }
 
+//------------------------------------------------------------------------------
+
+TEST(core_array, check_subspan_range)
+{
+  int m = 10;
+  int n = 3;
+  Array<int> arr(m);
+  ArrayView<int> arrv1(arr);
+  ArrayView<int> arrv2 = arrv1.subspan(n);
+  EXPECT_EQ(arrv2.size() + n, arrv1.size());
+  EXPECT_GE(&arrv2[0], &arr[0]);
```

Comment:
I think it would it be better to check that the address of the first element of arr2 is actually what we want it to be, specifically, EXPECT_EQ(&arrv2[0], &arr[n]).

Reply:
Agreed.